Tactical/Overhead.cpp

```diff
@@ -11092,6 +11034,105 @@ void HandleTurncoatAttempt( SOLDIERTYPE* pSoldier )
 	}
 }
 
+void EscapeTimerCallback()
+{
+    const bool chanceToEscape = Chance(75);
+    bool escaped = false;
+    // Look for an escape direction for remaining mercs
+    std::vector<WorldDirections> possibleEscapeDirections{ NORTH, EAST, SOUTH, WEST };
+    std::random_shuffle(possibleEscapeDirections.begin(), possibleEscapeDirections.end());

```

Comment:
random_shuffle is deprecated, use std::shuffle with a std::mt19937 generator instead

Reply:
Like this?
std::shuffle(possibleEscapeDirections.begin(), possibleEscapeDirections.end(), std::mt19937(time(nullptr)));

Reply:
https://en.cppreference.com/w/cpp/algorithm/random_shuffle see the examples